### PR TITLE
[REF-3868] Added method to retrieve field mappings

### PR DESCRIPTION
**Pull request by Chucheen**

### CODE REVIEW

**Comment by Chucheen** [comment]
[Expose Solovis field definitions to Synap](https://trello.com/c/i5jiNm2y/3868-expose-solovis-field-definitions-to-synap)

**Comment by Chucheen** [comment]
[Expose Solovis field definitions to Synap](https://trello.com/c/i5jiNm2y/3868-expose-solovis-field-definitions-to-synap)

**Comment by Chucheen** [comment]
[Expose Solovis field definitions to Synap](https://trello.com/c/i5jiNm2y/3868-expose-solovis-field-definitions-to-synap)

**Comment by Chucheen** [comment]
[Expose Solovis field definitions to Synap](https://trello.com/c/i5jiNm2y/3868-expose-solovis-field-definitions-to-synap)

**Comment by Chucheen** [comment]
[Expose Solovis field definitions to Synap](https://trello.com/c/i5jiNm2y/3868-expose-solovis-field-definitions-to-synap)

**Comment by Chucheen** [comment]
[Expose Solovis field definitions to Synap](https://trello.com/c/i5jiNm2y/3868-expose-solovis-field-definitions-to-synap)

File: lib/solovis_api_client/site.rb [hunk 1]
```diff
@@ -33,6 +33,10 @@ def set_field_value(property_definition_id, resource_id, new_value)
     end
   end
 
+  def get_field_mappings
```
Comment (rfleury2) [inline lib/solovis_api_client/site.rb]: This method needs a spec

**Comment by Chucheen** [comment]
@rfleury2 back to you! Thankyou!

**Pull request merged**